oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxLogicalVolumeGroup.java

```diff
@@ -0,0 +1,122 @@
+        // This requires elevated permissions and may fail
+        for (String s : ExecutingCommand.runNative("pvs -o vg_name,pv_name")) {
+            String[] split = ParseUtil.whitespaces.split(s.trim());
+            if (split.length == 2 && split[1].startsWith("/dev/")) {
+                physicalVolumesMap.computeIfAbsent(split[0], k -> new HashSet<>()).add(split[1]);
+            }
+        }
+
+        // Populate lv map from udev
+        Udev.UdevContext udev = Udev.INSTANCE.udev_new();
+        try {
+            Udev.UdevEnumerate enumerate = udev.enumerateNew();
+            try {
+                enumerate.addMatchSubsystem(BLOCK);
+                enumerate.scanDevices();
+                for (Udev.UdevListEntry entry = enumerate.getListEntry(); entry != null; entry = entry.getNext()) {
+                    String syspath = entry.getName();
+                    Udev.UdevDevice device = udev.deviceNewFromSyspath(syspath);
+                    if (device != null) {
+                        try {
+                            String devnode = device.getDevnode();
+                            if (devnode != null && devnode.startsWith("/dev/dm")) {
+                                String uuid = device.getPropertyValue(DM_UUID);
+                                if (uuid != null && uuid.startsWith("LVM-")) {
+                                    String vgName = device.getPropertyValue(DM_VG_NAME);
+                                    String lvName = device.getPropertyValue(DM_LV_NAME);
+                                    if (!Util.isBlank(vgName) && !Util.isBlank(lvName)) {
+                                        logicalVolumesMap.computeIfAbsent(vgName, k -> new HashMap<>());
+                                        Map<String, List<String>> lvMapForGroup = logicalVolumesMap.get(vgName);
+                                        // Backup to add to pv set if pvs command failed
+                                        physicalVolumesMap.computeIfAbsent(vgName, k -> new HashSet<>());
+                                        Set<String> pvSetForGroup = physicalVolumesMap.get(vgName);
+
+                                        File slavesDir = new File(syspath + "/slaves");
+                                        File[] slaves = slavesDir.listFiles();
+                                        if (slaves != null) {
+                                            for (File f : slaves) {
+                                                String pvName = f.getName();
+                                                lvMapForGroup.computeIfAbsent(lvName, k -> new ArrayList<>())
+                                                        .add(pvName);
+                                                // Backup to add to pv set if pvs command failed
+                                                //Added /dev to remove duplicates like /dev/sda1 and sda1
+                                                pvSetForGroup.add("/dev/" + pvName);
```

Comment:
Either don't prepend `/dev/` here or prepend it to the `pvName` in the `lvMap` above.

Reply:
I have added `/dev` here because the `pvs -o vg_name,pv_name` adds values like `/dev/sda1` but the slaves directory adds only `sda1`.

Reply:
ok